src/views/RegionAdditionalInfo/styles.module.css

```
@@ -0,0 +1,32 @@
+.region-preparedness {
```

This should match the name of component
```suggestion
.region-additional-info {
```